builds/respec-worker.js

```diff
@@ -18,7 +18,7 @@ self.addEventListener("message", ({ data: originalData }) => {
       const { code } = data;
       const langs = data.languages.length ? data.languages : undefined;
```

Comment:
We need to probably want something like: 

```suggestion
      const langs = data.languages.length ? data.languages : ["text"];
```